src/app/Dashboard/DashboardLayoutToolbar.tsx

```diff
@@ -289,6 +356,57 @@ export const DashboardLayoutConfig: React.FunctionComponent<DashboardLayoutConfi
     [t, handleDeleteButton, currLayout.name]
   );
 
+  const dropdownItems = React.useMemo(() => {
+    return (
+      <DropdownList>
+        <DropdownItem key="template" itemId={0}>
+          {t('DashboardLayoutToolbar.SET_AS_TEMPLATE')}
+        </DropdownItem>
```

Comment:
Would it make sense to have another option called: `Download as template`?

The modal UI is the same. Just that the submit button becomes `Download` button that download the serialized template.

Reply:
I don't think we need to have a modal for the downloading, but I can put that back in.